src/abstractitensornetwork.jl

```diff
@@ -935,6 +935,31 @@ function add(tn1::AbstractITensorNetwork, tn2::AbstractITensorNetwork)
   return tn12
 end
 
+""" Scale each tensor of the network via a function vertex -> Number"""
+function scale!(
+  weight_function::Function, tn::AbstractITensorNetwork; verts=collect(vertices(tn))
```

Comment:
```suggestion
  weight_function::Function, tn::AbstractITensorNetwork; vertices=collect(vertices(tn))
```
Just biasing towards not using shorthands (and it matches the name of the function from Graphs.jl so will be easier to remember).